### PR TITLE
Fix Import of Verboselogs Package Dependency

### PR DESCRIPTION
**Pull request by dvonthenen**
## Proposed changes

This fixes the import of the verbose logs package to resolve the missing dependency. Verified broken in test/dev build `dev.1`.

## Types of changes

What types of changes does your code introduce to the community Python SDK?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update or tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I have lint'ed all of my code using repo standards
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

NA

## Summary by CodeRabbit

- **Refactor**
  - Updated import paths for the `verboselogs` module across multiple files to use relative imports. This change improves the maintainability and readability of the codebase.

### CODE REVIEW

File: deepgram/clients/live/v1/client.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 from websockets.sync.client import connect, ClientConnection
 import websockets
 
-from deepgram.utils import verboselogs
+from ....utils import verboselogs
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply deepgram/clients/live/v1/client.py:14]: Ignoring test cases and examples. I want to be able to import the "real" deepgram-sdk package to test these.

File: deepgram/__init__.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 __version__ = "0.0.0"
 
 # entry point for the deepgram python sdk
-from .client import verboselogs
+from .utils import *  # pylint: disable=wildcard-import
 from .client import Deepgram, DeepgramClient
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply deepgram/__init__.py:10]: There is an intentional ignore on this as we want to import everything.

File: deepgram/clients/analyze/v1/options.py [hunk 3]
```diff
@@ -2,13 +2,13 @@
...
 
-from typing import List, Union, Optional
 import logging
+from typing import List, Union, Optional
 
 from dataclasses import dataclass, field
 from dataclasses_json import config as dataclass_config, DataClassJsonMixin
 
-from deepgram.utils import verboselogs
+from ....utils import verboselogs
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply deepgram/clients/analyze/v1/options.py:11]: Will consider in a future version.

File: deepgram/__init__.py [hunk 4]
```diff
@@ -6,7 +6,7 @@
 __version__ = "0.0.0"
 
 # entry point for the deepgram python sdk
-from .client import verboselogs
+from .utils import *  # pylint: disable=wildcard-import
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply deepgram/__init__.py:9]: this is intentional for right now

File: deepgram/__init__.py [hunk 5]
```diff
@@ -6,7 +6,8 @@
 __version__ = "0.0.0"
 
 # entry point for the deepgram python sdk
-from .client import verboselogs
+import logging
+from .utils.verboselogs import *  # pylint: disable=wildcard-import
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply deepgram/__init__.py]: This is obsolete

File: deepgram/__init__.py [hunk 6]
```diff
@@ -6,7 +6,8 @@
 __version__ = "0.0.0"
 
 # entry point for the deepgram python sdk
-from .client import verboselogs
+import logging
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply deepgram/__init__.py:9]: This is obsolete

File: deepgram/__init__.py [hunk 7]
```diff
@@ -6,7 +6,8 @@
...
+from .utils.verboselogs import *  # pylint: disable=wildcard-import
 from .client import Deepgram, DeepgramClient
```
[the remark this answers is not in the document]
Comment (dvonthenen) [reply deepgram/__init__.py:25]: This is obsolete

**Review by jpvajda — APPROVED** [review]

**Pull request merged**